templates/bin_setup.erb

```diff
@@ -7,8 +7,8 @@
 set -e
 
 # Set up Ruby dependencies via Bundler
-gem list bundler --installed > /dev/null || gem install bundler
-bundle install
+command -v bundle > /dev/null || gem install bundler
```

Comment:
`command -v` is insufficient if you are an rbenv user. It will pass if you have rbenv installed in any ruby, as the shim for bundler will be in the path.